grunt/aliases.js

```diff
@@ -6,11 +6,7 @@ module.exports = {
 
   'production': [
     'clean',
-    'copy:pages',
-    'copy:modules', 
-    'copy:includes', 
-    'copy:images', 
-    'copy:fontcopy', 
+    'copy', 
```

Comment:
Are you sure this is right? This will run the copy:js and copy:dist tasks as well. The copy:js task should not be run on production as we handle the js through the bower_concat and then uglify tasks. The copy:dist task should be run right at the end? Correct me if I am wrong. Bit hard to see in github.